### PR TITLE
Testing 5

### PR DESCRIPTION
**Pull request by rahul799**
This PR will test the [second point](https://github.com/PecanProject/pecan/pull/2569#issuecomment-613412143)
2.Multiple rounds of human commits and re-invocations of the styler bot

### CODE REVIEW

File: models/dvmdostem/R/model2netcdf.dvmdostem.R [hunk 1]
```diff
@@ -227,81 +229,93 @@ model2netcdf.dvmdostem <- function(outdir, runstart, runend, pecan_requested_var
...
-                row.names = TRUE,
-                quote = FALSE)
+    write.table(sapply(ncout$var, function(x) {
+      x$longname
+    }),
+    file = file.path(outdir, paste0(as.character(lubridate::year(all_yrs[i])), ".nc.var")),
+    col.names = FALSE,
+    row.names = TRUE,
+    quote = FALSE
+    )
```
Comment (infotroph) [inline models/dvmdostem/R/model2netcdf.dvmdostem.R]: Here's a place I don't like the styler bot's output, but this looks like a corner case. Extending the test: I'll propose a change, accept it, then restyle and see what the bot does. My prediction: It reverts my change completely. My hope: It keeps it, possibly with minor whitespace adjustments.

```suggestion
    write.table(
      sapply(ncout$var, function(x) {x$longname}),
      file = file.path(outdir, paste0(as.character(lubridate::year(all_yrs[i])), ".nc.var")),
      col.names = FALSE,
      row.names = TRUE,
      quote = FALSE)
```

Comment (infotroph) [reply models/dvmdostem/R/model2netcdf.dvmdostem.R]: /style

**Comment by infotroph** [comment]
It worked! 👍 👍 👍  for @rahul799!
